drivers/base/bootvid/common.c

```diff
@@ -272,6 +323,21 @@ RleBitBlt(
 
 /* PUBLIC FUNCTIONS ***********************************************************/
 
+/*
+ * @implemented
+ */
+ULONG
+NTAPI
+VidSetTextColor(IN ULONG Color)
```

Comment:
And here:
https://git.reactos.org/?p=reactos.git;a=blob;f=drivers/base/bootvid/arm/bootvid.c;hb=592bafd60a33b82e62174f124ce80b2e0374a03a#l242

Reply:
Fixed.